test/fit_tests/deploy/os_ova_install.py

```diff
@@ -4,7 +4,11 @@
 Author(s):
 George Paulos
 
-This script installs blank OVA template.
+This script installs blank OVA templates into selected stack.
+if numvms is greater than 1, MAC address installation on OVA is done by stack number convention
```

Comment:
@gpaulos :  She was following what you already had set up as a pattern in our DHCP and DNS configuration in the lab.  For example:  stack14-ora.admin is 00:50:56:00:14:00